packages/vue/src/components/FCard/FCard.vue

```diff
@@ -2,7 +2,7 @@
     <div class="card card--default">
         <div v-if="hasHeaderSlot" class="card__header">
             <!--@slot Slot for the title. -->
-            <slot name="header" v-bind="{ headingSlotClass: headingClass }"></slot>
+            <slot name="header" class="card__header-label"></slot>
```

Comment:
```suggestion
            <slot name="header" v-bind="{ headingSlotClass: 'card__header-label' }"></slot>
```

- [ ] skicka med enligt förslag i scoped slot